### ISSUE TITLE
Problem: Missing flexible global RPC / chain selection

### ISSUE THREAD

**Issue by ricky321q** [cronos-labs/play-unreal-plugin#187]
For now developers have to manually set their cronos RPC and cronos ChainId,
I purpose that we should enable plugin server of selection,
developers could choose what's the env they're using(testnet / mainnet / devnet), then we have default rpc/chainId for testnet and mainnet, they don't have to change for it. if developers choosing devnet, then we let them to input like 8545 , 3000.
So developers don't have to set for every Actors and don't have to worry about wrong RPC / ChainId.
But also need to consider like our public RPC might not be powerful enough , they might want to use their own rpc to run.
So we could let developers to choose the network first, and we have option to still let them to configure about it.

**Comment by tomtau** [comment]
> But also need to consider like our public RPC might not be powerful enough , they might want to use their own rpc to run.

Yeah, I think this is the important point; it can perhaps have a default node for testing, but it should display warnings and point developers to blockdaemon, rockx or other node providers, so that they setup dedicated full nodes for production
